platform/src/components/aws/cognito-user-pool.ts

```diff
@@ -543,11 +543,9 @@ export class CognitoUserPool extends Component implements Link.Linkable {
             ),
             smsAuthenticationMessage: args.smsAuthenticationMessage,
             smsConfiguration: args.sms,
-            softwareTokenMfaConfiguration: output(args.softwareToken).apply(
-              (v) => ({
-                enabled: v ?? false,
-              }),
-            ),
+            softwareTokenMfaConfiguration: args.softwareToken
```

Comment:
`args.softwareToken` is of type `Input<boolean`.

This means that if it is set as either `true` or `false` it would be truthy. 

In this case seems to me that `enabled` should be set to softwareToken, like so:

`output(args.softwareToken).apply((token) => ({ enabled: token }))`